### PR TITLE
fix(FEC-8924): [V3][iOS] when touch and dragging from player area the page is not scrolling

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Move the prevent default from `touchstart` to `touchend` to enable scrolling on player 

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/overlay-action/overlay-action.js [hunk 1]
```diff
@@ -257,10 +259,11 @@ class OverlayAction extends BaseComponent {
     return (
       <div
         className={`${style.overlayAction} ${this.state.animation ? style.in : ''}`}
-        onMouseDown={e => this.onOverlayPointerDown(e)}
-        onTouchStart={e => this.onOverlayPointerDown(e)}
-        onMouseUp={e => this.onOverlayMouseUp(e)}
-        onTouchEnd={e => this.onOverlayTouchEnd(e)}>
+        onMouseDown={this.props.isVr && this.props.forceScrollInVideoPlayer ? e => this.onOverlayPointerDown(e) : null}
```
[the remark this answers is not in the document]
Comment (Yuvalke) [reply src/components/overlay-action/overlay-action.js]: this.props.isVr would be able to change on every loadMedia and changed by event

File: src/components/overlay-action/overlay-action.js [hunk 2]
```diff
@@ -18,7 +18,9 @@ const mapStateToProps = state => ({
   isPlayingAdOrPlayback: isPlayingAdOrPlayback(state.engine),
   iconType: state.overlayAction.iconType,
   playerHover: state.shell.playerHover,
-  isMobile: state.shell.isMobile
+  isMobile: state.shell.isMobile,
+  isVr: state.engine.isVr,
+  forceScrollInVideoPlayer: state.config.forceScrollInVideoPlayer
```
[the remark this answers is not in the document]
Comment (Yuvalke) [reply src/components/overlay-action/overlay-action.js]: added

File: src/components/overlay-action/overlay-action.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (yairans) [reply src/components/overlay-action/overlay-action.js]: The UI also updating in every loadMedia

File: src/components/overlay-action/overlay-action.js [hunk 3]
```diff
@@ -156,6 +158,22 @@ class OverlayAction extends BaseComponent {
...
+   * handler for click for regular videos - (not VR)
+   *
+   * @returns {void}
+   * @memberof OverlayAction
+   */
+  onOverlayClick(): void {
+    if (!this.props.isMobile) {
+      this.overlayClick();
+    } else {
+      if (this.props.playerHover) {
```
Comment (yairans) [inline src/components/overlay-action/overlay-action.js]: Seeks like `onClick` emitted twice on mobile. Look at https://github.com/kaltura/playkit-js-ui/pull/188

File: src/components/overlay-action/overlay-action.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (yairans) [reply src/components/overlay-action/overlay-action.js]: example https://github.com/kaltura/kaltura-player-js/blob/60edaf6c916979ec7ca28f7a09403642ee0af8b0/src/kaltura-player.js#L88

**Pull request merged**